src/zope/app/security/globalmodules.zcml

```diff
@@ -487,7 +487,15 @@
     </module>
     <module module="multifile">
       <allow attributes="MultiFile" />
-    </module>>
+    </module>
   </configure>
 
+  <!-- The following package has been deprecated in Python 3.4.
+       Let's use a similar trick of detecting whether the package
+       "statistics" is available, as it was added in 3.4. -->
```

Comment:
There's a [backport of statistics](https://pypi.org/project/statistics/#description) for Python 2.7.  Maybe `asyncio` would be a choice less likely to conflict? The only `asyncio` backport I know of is `trollius`.

Reply:
Yeah, and ZEO depends on `trollius`, so it's likely to be present in a Zope 3 application.

Unless.... trollius doesn't actually ship an `asyncio` package, it ships a `trollius`, so the check should still work.